src/sssom/sparql_util.py

```diff
@@ -26,13 +28,16 @@ class EndpointConfig:
     predmap: Dict[str, str]
     predicates: Optional[List[str]]
     limit: Optional[int]
-    prefix_map: Optional[Dict[str, str]]
     include_object_labels: bool = False
+    prefix_map: Dict[str, str] = field(default_factory=dict)
 
 
 def query_mappings(config: EndpointConfig) -> MappingSetDataFrame:
     """Query a SPARQL endpoint to obtain a set of mappings."""
-    sparql = SPARQLWrapper(config.url)
+    if not config.prefix_map:
+        raise TypeError
```

Comment:
Maybe a more expressive error message here

Reply:
Done in 1c25c94, but I inherited this not having a useful error message from before.